edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/metrics/AvaliabilityMetrics.cs

```diff
@@ -94,16 +102,32 @@ public void ComputeAvailability(ModuleSet desired, ModuleSet current)
             }
         }
 
+        public void Dispose()
+        {
+            this.checkpoint.Dispose();
+        }
+
+        public void OnCleanShutdown()
+        {
+            try
+            {
+                File.Delete("shutdown_time");
```

Comment:
can you add comments for what "shutdown_time" store? in what format? and when it create/update?